### PR TITLE
G3 tic-tac-toe

### PR DESCRIPTION
**Pull request by Psy-Kosh**

### CODE REVIEW

File: tic_tac_toe/game.py [hunk 1]
```diff
@@ -33,7 +38,12 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    for row in board:
+        for space in row:
```
Comment (seg2151) [inline tic_tac_toe/game.py:42]: here you can just check if '-' in row, no need to iterate over each item in the row

File: tic_tac_toe/game.py [hunk 2]
```diff
@@ -47,7 +57,7 @@ def _is_winning_combination(board, combination, player):
     Returns True of all three positions in the combination belongs to given
     player, False otherwise.
     """
-    pass
+    return all(board[row][col] == player for row,col in combination)
```
Comment (seg2151) [inline tic_tac_toe/game.py:61]: nice! y'all made this super clean

File: tic_tac_toe/game.py (hunk 1 above)
Comment (Psy-Kosh) [reply tic_tac_toe/game.py:42]: Whoops, should have thought of that. Thanks.

File: tic_tac_toe/game.py (hunk 2 above)
Comment (Psy-Kosh) [reply tic_tac_toe/game.py:61]: Aww, thank you. :)

**Pull request closed without merging**